packages/transport/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.1.17
+
+-   fix(connect): content type header applied for native (#9594) (c84270e13)
+-   fix(connect): use type header with the bridge API (#9590) (b8bb2dfe6)
```

Comment:
I'd change this to just a single line probably, ale having `connect` in commit message triggers my OCD slightly. maybe it doesnt matter. I am not sure

Reply:
I understand the OCD with that. The commits are actually changing only transport but the commit message is a bit confusing, I am changing it.